libs/domain/picking/src/components/picking-list-item/picking-list-item.component.ts

```diff
@@ -100,8 +92,17 @@ export class PickingListItemComponent
           `
         )}
 
-        <oryx-button slot="footer" type=${ButtonType.Primary} size=${Size.Lg}>
-          <button ?disabled=${this.isDisabled} @click=${this.startPicking}>
+        <oryx-button
+          slot="footer"
+          type=${ButtonType.Primary}
+          size=${Size.Lg}
+          ?loading=${this.upcomingPickingListId === this.pickingList.id}
+        >
+          <button
+            ?disabled=${this.upcomingPickingListId &&
```

Comment:
Should we disable only other picking lists but not that one that proceed picking? Maybe just:

`?disabled=${!!this.upcomingPickingListId}`

Reply:
As I said before, the loading state makes the button inactive, so the user can't interact with it.